packages/drivers/odsp-driver/src/odspSummaryUploadManager.ts

```diff
@@ -81,7 +81,7 @@ export class OdspSummaryUploadManager {
 		tree: api.ISummaryTree,
 	): Promise<IWriteSummaryResponse> {
 		const containsProtocolTree = isCombinedAppAndProtocolSummary(tree);
-		const { snapshotTree, blobs } = await this.convertSummaryToSnapshotTree(
+		const { snapshotTree, blobs } = this.convertSummaryToSnapshotTree(
```

Comment:
i'd prefer to make this change in a separate PR. Seems fine, but unrelated to tree2 snapshots and if there are any details impacted by JS turn order (I sure hope not) it could be observable.